### DISCUSSION TITLE
Download failure on Zonia.com

### DISCUSSION THREAD

**Discussion by tombarnettnh** [aclap-dev/video-downloadhelper#763] [Help and Q&A]
### Your question

Failure to download

### Addon & CoApp version and details

Using Version 6.5.1

Bug details:
{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://zonia.com/is-episode-1-live-4903467a",
  "error": {
    "details": "SideDownload error: : [dash @ 0000019224f68940] Error when loading first fragment of playlist\r\n[in#0 @ 0000019224f68780] Error opening input: Server returned 404 Not Found\r\nError opening input file https://video-node.swarmcdn.com/f78b4e69-52cb-4c35-bd18-700668a55ad8/52e7c3256d3f4f13ea09cd183b8153b80e388be0f0c70ba2b33cbcff77de08e5.mpd?t=0.9697234040326956.\r\nError opening input files: Server returned 404 Not Found\r\n",
    "downloadable_id": "downloadable_5532604301449634",
    "error": "coapp_failure",
    "id": "coapp_failure_44a934c9-7a84-49a3-8861-4eef4e13598c"
  }
}

Thanks.

**Comment by Mattmjs** [comment]
Duplicate of #732
